programs/native/bpf_loader/src/lib.rs

```diff
@@ -7,6 +7,7 @@ extern crate env_logger;
 extern crate log;
 extern crate libc;
 extern crate solana_rbpf;
+#[macro_use]
```

Comment:
I thought this went away in a recent release of rust, Is macro_use still required?

Reply:
Removing it produces:
```
error: cannot find macro `solana_entrypoint!` in this scope                                                                                        
   --> programs/native/bpf_loader/src/lib.rs:142:1                                                                                                 
    |                                                                                                                                              
142 | solana_entrypoint!(entrypoint);                                                                                                              
    | ^^^^^^^^^^^^^^^^^                 
```